### PR TITLE
Introduce CI, + other improvements

### PR DESCRIPTION
**Pull request by andife**
* Use project structure based on cookiecutter

* Add flake8 to CI pipline

* Add codecov to CI pipline 

* Reduce spelling errors

* Remove pytest cache from version control.

* Use github actions for CI

* Use setup.cfg as central configuration replacing requirements and setup.py

* Adjust tox.ini to new setup.cfg.

* Fix syntax error in gh yaml.

* Add badge and rename workflow name

* Ignore github actions if [skip ci] is contained in commit message

* Fix incorrect badge path in README [skip-ci]

* Remove obsolete version module.

* Move test data into subfolder and refactor conftest module.

* Add init module to test dir to allow coverage in local dev with pytest.

* Replace os.path with pathlib.Path. Convert `dumb_res` to fixture.

### CODE REVIEW

**Review by olivertomic — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributing to hoggorm
+
+If you are interested in contributing to hoggorm, your contributions will fall into two categories:
+
+1. You want to implement a new feature:
+....
+2. You want to fix a bug:
```
Comment (olivertomic) [inline CONTRIBUTING.md:7]: We should introduce a third category: Improvement of documentation.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    - Feel free to send a Pull Request any time you encounter a bug. Please provide a clear and concise description of what the bug was. If you are unsure about if this is a bug at all or how to fix, post about it in an issue.
+
+Once you finish implementing a feature or bug-fix, please send a Pull Request to https://github.com/olivertomic/hoggorm
+
+## Developing hoggorm
+
+To develop hoggorm on your machine, here are some tips:
+
+1. Create an enviroment for hoggorm
```
Comment (olivertomic) [inline CONTRIBUTING.md:16]: Little typo: environment

File: setup.cfg [hunk 3]
```diff
@@ -1,24 +1,25 @@
 [metadata]
 name = hoggorm
-version = attr: hoggorm.__version__
-description = Package for explorative multivariate statistics
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-keywords =
+version=0.13.3
+description = Package for explorative multivariate statistics.
+author = Oliver Tomic
```
Comment (olivertomic) [inline setup.cfg:5]: I guess it is possible to insert multiple authors? Other important contributors are Kristian Hovde Liland, Thomas Graff and Andreas Fehlner.

**Review by olivertomic — APPROVED** [review]
A few small easy to fix suggestions.

File: CONTRIBUTING.md (hunk 2 above)
Comment (andife) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 1 above)
Comment (andife) [reply CONTRIBUTING.md:7]: Done

File: setup.cfg (hunk 3 above)
Comment (andife) [reply setup.cfg:5]: It seems that there is no standard way?
https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py

One could add an AUTHORS-File like 
https://github.com/cookiecutter/cookiecutter/blob/master/AUTHORS.md

Comment (olivertomic) [reply setup.cfg:5]: Authors.md looks like a nice way giving all authors credit. Let's implement this.

**Pull request merged**